linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_ownership/rule.yml

```diff
@@ -29,6 +29,7 @@ identifiers:
 references:
     anssi@debian8: NT28(R46),NT28(R5)
     anssi@debian9: NT28(R46),NT28(R5)
+    anssi@debian10: NT28(R46),NT28(R5)
```

Comment:
Since all the anssi mappings are the same, please collapse into a single entry e.g. `anssi: NT28(R46),NT28(R5)`